python/pyarrow/tests/test_extension_type.py

```diff
@@ -267,6 +267,32 @@ def test_ext_type__storage_type():
     assert ty.__class__ is ParamExtType
 
 
+def test_ext_type_byte_width():
+    # Test for fixed-size binary types
+    ty = UuidType()
+    assert ty.storage_type.byte_width == 16
+    ty = ParamExtType(5)
+    assert ty.storage_type.byte_width == 5
```

Comment:
```suggestion
    assert ty.byte_width == 16
    ty = ParamExtType(5)
    assert ty.byte_width == 5
```

All those tests should check the extension type itself, and not trough `.storage_type`, because the goal of the PR is to make those attributes available on the extension type itself?